package.json

```diff
@@ -52,7 +52,7 @@
     "less-plugin-autoprefix": "1.5.1",
     "less-plugin-clean-css": "1.5.1",
     "nomnom": "1.8.1",
-    "openlayers": "fredj/ol3#afb2084",
+    "openlayers": "fredj/ol3#b49546d",
```

Comment:
Yes, but commit https://github.com/fredj/ol3/commit/b49546d is in branch https://github.com/fredj/ol3/tree/modify_condition which includes the change. :)